import_export_celery/admin_actions.py

```diff
@@ -39,18 +39,28 @@ def run_export_job_action(modeladmin, request, queryset):
 
 
 def create_export_job_action(modeladmin, request, queryset):
-    if queryset:
+    selected = request.POST.getlist('_selected_action')
```

Comment:
What is this, why is it here?

Reply:
when I am trying to export 10 million of rows, system got hanged. so I do not want to use queryset without the below statement. due to this reason. i used that code
queryset.query.select_related =False